setup.py

```diff
@@ -23,7 +23,7 @@ def package_data(pkg, root_list):
 
 setup(
     name='xblock-drag-and-drop-v2',
-    version='2.0.15',
+    version='2.0.16',
```

Comment:
Note: this was added to be able to build on to the sandbox and might be removed into another PR when we are ready to release